eo-runtime/src/test/java/EOorg/EOeolang/EOcageTest.java

```diff
@@ -78,21 +66,6 @@ void checksThatEmptyCageHasIdentity() {
         );
     }
 
-    @Test
-    @Disabled
-    void writesItselfToItself() {
```

Comment:
@maxonfjvipon Same here. I don't quite understand why this test was removed.

Reply:
@maxonfjvipon I believe that the idea of testing the `cage` object by writing itself to itself looks good. Perhaps instead of taking the refactoring of this test literally, we could change the logic to accommodate the new functionality of the `cage` object using this idea. How do you think is it possible?